ci/Makefile

```diff
@@ -2,7 +2,17 @@
 .PHONY: setup-conda-env build-hail-ci push-hail-ci test-locally
 .PHONY: run-service deploy
 
-PROJECT = $(shell gcloud config get-value project)
+PROJECT=${shell gcloud config get-value project}
+
+# Get unique instance name, external_ip, and zone, to avoid hardcoding vars
+# There may be a better way of doing this, but simply ${ARR[N]} did not work
+# 1) Get a unique instance name, either based on passed instance name, passed tag name,
+# or by choosing first listed instance
+# 2) Based on that instance, get EXTERNAL_IP and ZONE, to avoid hardcoding
+# TODO: This behavior (choosing first instance if no instance specified) may be undesirable
```

Comment:
I think this should be an error. If we connect to a machine that hasn't had it's sshd config modified to permit listening on the machine's public IP, nothing will work and it's not obvious why.

Reply:
Sounds good